src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -730,7 +728,10 @@ public void SendAsync_NullRequest_ThrowsArgumentNullException()
 
                     using (MyHandler handler = new MyHandler())
                     {
-                        Assert.ThrowsAsync<ArgumentNullException>(() => handler.SendAsync(null)).Wait();
+                        // Getting the Task first from the .SendAsync() call tests
+                        // that the exception comes from the async Task path.
```

Comment:
Does the previous pattern not able to observe the `ArgumentNullException`? (I asked because the `ActiveIssue` label is removed)

Reply:
Suprisingly, using the previous Assert.ThrowsAsync pattern causes the test to fail with:

>Expected: typeof(System.ArgumentNullException)
Actual:   typeof(System.Net.Http.HttpRequestException): An error occurred while sending the request.) ---> Xunit.Sdk.ThrowsException: Assert.Throws() Failure

So, by changing it as I've done the test passes everywhere.  And it also makes it clearer about what is being tested.

Reply:
Actually, in re-looking at this change here, I need to change it.  The test is now passing because the Assert.ThrowsAsync() is no longer being awaited!!  When it is, it is still throwing a different exception on UAP version.  I will submit a commit to fix this.